.devcontainer/devcontainer.json

```diff
@@ -4,7 +4,7 @@
     "dockerfile": "Dockerfile",
     "context": ".."
   },
-  "appPort": 8123,
+  "appPort": 8124,
```

Comment:
Isn't this was the supervisor container uses? :P

Reply:
Supervisor uses 9123